src/packages/api/bill/bill-api.ts

```
@@ -0,0 +1,108 @@
+			this.apiClient.environment === Environment.Production ? 'quickbooks.api.intuit.com' : 'sandbox-quickbooks.api.intuit.com';
+
+		// Return the Company Endpoint
+		return `https://${environmentUrl}/v3/company/${token.realmId}`;
+	}
+
+	/**
+	 * Format the Response
+	 * @param response - The Response
+	 * @returns The Bills
+	 */
+	protected formatResponse(response: any): Array<Bill> {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)